adapter/test_slack.py

```diff
@@ -1,7 +1,7 @@
 import pytest
```

Comment:
Do we need to create tests that consider missing metadata?

Reply:
Yes, it seems necessary.  Nice!!, Think of it i also need to  handle cases where metadata is missing or doesn't match expectations.